### PR TITLE
feat: add customisation option for document classification

### PR DESCRIPTION
**Pull request by elliotwutingfeng**
Classification footer can be changed to anything other than "Confidential" (limited to 15 characters or less), like "Top Secret", "Restricted" etc.

Reset to Default button is also provided.

### CODE REVIEW

File: src/components/ContentEditor.vue [hunk 1]
```diff
@@ -61,24 +73,31 @@ export default {
...
         this.$emit('update:recno', this.editedRecno)
+        this.$emit('update:classification', this.editedClassification)
         this.$emit('update:isEditorActive', false)
         this.$router.push('/edit')
 
         // Save edit
         if (localStorageAvailable()) {
           localStorage.setItem(FBC_RECORD_CONTENT, this.editedContent)
           localStorage.setItem(FBC_RECORD_NUMBER, this.recno)
+          localStorage.setItem(FBC_CLASSIFICATION, this.classification)
```
Comment (louh) [inline src/components/ContentEditor.vue]: There seems to be a bug where this doesn't get saved properly, and neither does the `recno`. Setting the item to `this.editedClassification` and `this.editedRecno` (fixing the previous line) should make this work.

File: src/components/ContentEditor.vue [hunk 2]
```diff
@@ -193,6 +219,46 @@ export default {
   margin-bottom: 0;
 }
 
+.classification-input {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
```
Comment (louh) [inline src/components/ContentEditor.vue:227]: We also need a margin between the classification input line and the record number line above. A value of `1em` should be fine.

**Review by louh — CHANGES_REQUESTED** [review]
Hi @elliotwutingfeng, thank you for your contribution! I'm happy to take this. Just a couple minor changes: one bug fix (including a fix for an existing bug that you have uncovered, thank you), and a style change. If you're able to update your PR I will approve and merge. Thanks!

File: src/components/ContentEditor.vue (hunk 1 above)
Comment (elliotwutingfeng) [reply src/components/ContentEditor.vue]: Fixed!

File: src/components/ContentEditor.vue (hunk 2 above)
Comment (elliotwutingfeng) [reply src/components/ContentEditor.vue:227]: Fixed!

**Comment by louh** [comment]
Look great. Thanks for the addition, @elliotwutingfeng!

**Review by louh — APPROVED** [review]

**Pull request merged**